coreos-base/coreos/coreos-0.0.1.ebuild

```diff
@@ -69,11 +69,12 @@ RDEPEND=">=sys-apps/baselayout-3.0.0"
 # If protocol 1 is installed it must be configured to provide the default
 # implementation based on whether protocol 2 is enabled or not.
 RDEPEND="${RDEPEND}
-	etcd_protocols_1? (
-		dev-db/etcd:1
-		!etcd_protocols_2? ( dev-db/etcdctl )
-	)
-	etcd_protocols_2? ( dev-db/etcd:2 )
+	etcd_protocols_1? ( dev-db/etcd:1 )
+	etcd_protocols_2? (
+        dev-db/etcd:2
+        app-admin/etcd-wrapper:2
```

Comment:
Typically [portage profiles](https://wiki.gentoo.org/wiki/Profile_(Portage)) are used for arch specific packages. It looks like the SDK makes no use of profiles at all, so my comment is not really applicable.